# Execute previously planned gripper movement

---
bool success
